src/main/java/lotto/domain/LottoFactory.java

```diff
@@ -0,0 +1,51 @@
+package lotto.domain;
+
+import java.util.*;
+
+public class LottoFactory {
+
+    private static final int LOTTO_MIN = 1;
+    private static final int LOTTO_MAX = 45;
+    private static final int LOTTO_SIZE = 6;
+    private static final int LOTTO_PRICE = 1000;
+    private static final String LOTTO_AMOUNT_PATTERN = "^[1-9]\\d+\\s*원";
+    private static final String NON_DIGIT_REGEX = "[^0-9]";
+
+
+    public static List<Lotto> issueLottos(String purchaseAmount) {
+
+        List<Lotto> lottos = new ArrayList<>();
+
+        for (int i = 0; i < calculateLottoCount(purchaseAmount); i++) {
+            lottos.add(issueLotto());
+        }
+
+        return lottos;
+    }
+
+    private static int calculateLottoCount(String purchaseAmount) {
```

Comment:
```suggestion
    private static int calculateLottoCount(int purchaseAmount) {
```
금액은 숫자니까 숫자타입만 받아본다면 불필요한 로직도 많이 사라질 것 같아요
입력금액이 숫자인지 검증하는 정도는 입력View 영역에서 처리할 수 있지 않을까요?

Reply:
View 영역은 입력받는 역할만 해야할 줄 알았는데 입력받은 값 검증까지 하는게 맞는거같아 수정했습니댜!